Vue/vaahtwo/pages/public/Signup.vue

```diff
@@ -11,18 +11,21 @@ const root = useRootStore();
 const auth = useAuthStore();
 const route = useRoute();
 
+
 onMounted(async () => {
+
     document.title = 'Sign Up';
     root.showResponse(route.query);
     auth.verifyInstallStatus();
     await root.getAssets();
-});
+    await root.checkSignupPageVisible();
```

Comment:
@we-anil-y001 that should be inside the `onMounted`

Reply:
ok sir

Reply:
sir this is already in onMounted